frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import * as plugins from '../../../../../../../public/plugins';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../../../../../../public/components/dashboard/dashboard-card';
+import { CapacityBody, CapacityItem } from '../../../../../../../public/components/dashboard/capacity-card';
+import { withDashboardResources, DashboardItemProps } from '../../../../../../../public/components/dashboards-page/with-dashboard-resources';
+import { humanizeBinaryBytesWithoutB } from '../../../../../../../public/components/utils';
+import { getInstantVectorStats, getRangeVectorStats, GetStats } from '../../../../../../../public/components/graphs/utils';
+import { CAPACITY_CARD_QUERIES } from '../../../../constants/storage';
+
+const defaultQueries = {
+  [CAPACITY_CARD_QUERIES.CAPACITY_TOTAL]: 'sum(node_filesystem_size_bytes)',
+  [CAPACITY_CARD_QUERIES.CAPACITY_USED]: '(sum(node_filesystem_size_bytes) - sum(node_filesystem_free_bytes))[60m:5m]',
+};
+
+const getLastStats = (response, getStats: GetStats): React.ReactText => {
+  const stats = getStats(response);
+  return stats.length > 0 ? stats[stats.length - 1].y : null;
+};
+
+const getQueries = () => {
```

Comment:
We don't require this as we are not calling extensions in storage capacity.

Reply:
Ack, updated the PR